opennms-webapp/src/main/webapp/admin/sched-outages/index.jsp

```diff
@@ -68,8 +68,8 @@
 	WriteableThreshdDao threshdDao = BeanUtils.getBean("thresholdingContext", "threshdDao", WriteableThreshdDao.class);
 	WriteablePollOutagesDao pollOutagesDao = BeanUtils.getBean("pollerConfigContext", "pollOutagesDao",
 			WriteablePollOutagesDao.class);
-
-	NotifdConfigFactory.init(); //Must do this early on - if it fails, then just throw the exception to the web gui
+    NotifdConfigFactory.init();
+	NotifdConfigFactory.getInstance().reload(); //Must do this early on - if it fails, then just throw the exception to the web gui
```

Comment:
same as above. Also please fix your indent. One is tab and one is space.

Reply:
Fixed